app/posts/releasing-rails-libraries-for-version-5.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Releasing the Rails libraries for version 5 of the GOV.UK Design System
+description: Notes on how the Rails libraries for the GOV.UK Design System are developed and the work required to support version 5
+date: 2024-01-10
+author:
+  name: Peter Yates
+  url: https://github.com/peteryates
+---
+
+Version 5 of the GOV.UK Design System was released on December 8th 2023. Half an hour later, we released version 5 of the x-govuk [form builder](https://govuk-form-builder.netlify.app/) and [components](https://govuk-components.netlify.app/) libraries.
+
+The short turnaround time might suggest it was a quick job, but work on the included features started in May and we've made some significant improvements since then.
```

Comment:
Smart quote for smart people. (There are likely others that need updating)

```suggestion
The short turnaround time might suggest it was a quick job, but work on the included features started in May and we’ve made some significant improvements since then.
```

Reply:
Why update them when the build progress automatically fixes them (thanks `smartypants`!)

Reply:
It’s a good question, and touches on an existential crisis. Perhaps we leave these and stop trying to catch issues in PRs (and I can update previous posts to use ASCII to satisfy my OCD needs)

Reply:
@paulrobertlloyd that really is OCD! Maybe we should follow the principle of being lax about input (curly or straight quotes) and strict about output...? 😄

Reply:
I’ve really got a lot of issues…

![Scene from ‘I’m Alan Partidge’ where he responds with: “yeah, of What Car? Magazine“."](https://img.gifglobe.com/grabs/partridgecloud/S02E06/gif/vPCVHa3Qnqqr.gif)